multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/auditlogging/AuditLogBean.java

```diff
@@ -1,77 +1,52 @@
 package org.cloudfoundry.multiapps.controller.core.auditlogging;
 
-import org.cloudfoundry.multiapps.controller.core.auditlogging.impl.AuditLoggingFacadeSLImpl;
-import org.cloudfoundry.multiapps.controller.core.util.UserInfo;
-import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.DependsOn;
-import org.springframework.security.core.Authentication;
-import org.springframework.security.core.context.SecurityContextHolder;
-import org.springframework.security.oauth2.client.authentication.OAuth2AuthenticationToken;
 
 import javax.inject.Inject;
-import javax.sql.DataSource;
-
-import static org.cloudfoundry.multiapps.controller.core.util.SecurityUtil.USER_INFO;
 
 @Configuration
 public class AuditLogBean {
 
-    @Inject
-    @Qualifier("dataSource")
-    protected DataSource dataSource;
-
     @Bean
-    public AuditLoggingFacade buildAuditLoggingFacade() {
```

Comment:
How `AuditLoggingFacade` will be initialised as bean?

Reply:
I will be initialized in the xsa

Reply:
Okay but oss project must be also start and run without xsa part. In this way now it will always fails to initialize other beans due to missing `AuditLoggingFacade` bean initialization. Think for a way how to initialize this interface in oss and xsa project in the same time and override initialization in xsa project.